war/src/main/webapp/css/style.css

```diff
@@ -1859,6 +1859,18 @@ table#legend-table td {
     border-color: #faebcc;
 }
 
+body.no-decoration #main-panel {
+    margin: 0 auto !important;
+}
+body.no-decoration #page-head,
+body.no-decoration #side-panel,
+body.no-decoration footer {
+    display: none;
+}
```

Comment:
All of this will be removed when [JENKINS-34670](https://issues.jenkins-ci.org/browse/JENKINS-34670) is implemented.